src/Components/test/testassets/BasicTestApp/Program.cs

```diff
@@ -28,16 +27,18 @@ public static async Task Main(string[] args)
 
             var builder = WebAssemblyHostBuilder.CreateDefault(args);
 
+            var httpClient = new HttpClient();
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY")))
             {
                 // Needed because the test server runs on a different port than the client app,
                 // and we want to test sending/receiving cookies under this config
-                WebAssemblyHttpMessageHandlerOptions.DefaultCredentials = FetchCredentialsOption.Include;
+                httpClient = new HttpClient(new ConfigureForCorsHandler(new WebAssemblyHttpHandler()));
```

Comment:
Now we have the capability of doing this on a per request basis, I would have a slight preference for moving the CORS-specific test logic into the CORS-specific test and not having it apply globally to the whole project.

Do you think that's feasible?